controllers/mutation_reconcile_looper.go

```diff
@@ -801,19 +804,47 @@ func (m *MutationReconcileLooper) getValues(ctx context.Context, obj *v1alpha1.M
 			return nil, fmt.Errorf("could not read values file: %w", err)
 		}
 
-		var data map[string]any
 		if err := yaml.Unmarshal(dataFile, &data); err != nil {
 			return nil, fmt.Errorf("failed to unmarshal values: %w", err)
 		}
-
 		var found bool
 		if obj.ValuesFrom.FluxSource.SubPath != "" {
 			data, found = extractSubpath(data, obj.ValuesFrom.FluxSource.SubPath)
 			if !found {
 				return nil, errors.New("subPath not found")
 			}
 		}
+	} else if obj.ValuesFrom.ConfigMapSource != nil {
+		cm := &corev1.ConfigMap{}
+		key := types.NamespacedName{
+			Name:      obj.ValuesFrom.ConfigMapSource.SourceRef.Name,
+			Namespace: namespace,
```

Comment:
In case of MPAS this would be the Project namespace. Is that where the ConfigMap would reside? Is it possible it's gonna be in a different namespace?

Reply:
I think the values.yam is for a product for a given project. So it has to reside in the project ns.

Reply:
Sounds good!